swtbot_test/src/kerneltestsuites/kernelCCRXKernelGUI.java

```diff
@@ -0,0 +1,378 @@
+import parameters.ProjectParameters.MenuName;
+import parameters.ProjectParameters.RTOSApplication;
+import parameters.ProjectParameters.RTOSType;
+import parameters.ProjectParameters.RTOSVersion;
+import parameters.ProjectParameters.TargetBoard;
+import platform.PlatformModel;
+import utilities.PGUtility;
+import utilities.Utility;
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class kernelCCRXKernelGUI {
+	private static SWTWorkbenchBot bot;
+	private static ProjectModel projectModelSpecific = new ProjectModel();
+	private static final String PLATFORM_XML_FILE = "xml/platformdata.xml";
+	private static final String RTOS_PG_XML_FILE = "xml/rtospg.xml";
+
+	@BeforeClass
+	public static void beforeClass() throws Exception {
+		bot = new SWTWorkbenchBot();
+		PlatformModel.loadPlatformModel(new File(Utility.getBundlePath(LogUtil.PLUGIN_ID, PLATFORM_XML_FILE)));
+		RTOSManager.loadRTOSModel(new File(Utility.getBundlePath(LogUtil.PLUGIN_ID, RTOS_PG_XML_FILE)));
+		projectModelSpecific = PGUtility.prepareProjectModel(RTOSType.FREERTOSKERNEL, RTOSVersion.Kernel_1_0_7,
+				RTOSApplication.KERNEL_BARE, Constants.CCRX_TOOLCHAIN, TargetBoard.BOARD_RSK_RX65N_2MB);
+	}
+
+	@Test
+	public void tc_01_CreateKernelProject() throws Exception {
+		PGUtility.createProject(RTOSType.FREERTOSKERNEL, RTOSVersion.Kernel_1_0_7, RTOSApplication.KERNEL_BARE,
+				Constants.CCRX_TOOLCHAIN, TargetBoard.BOARD_RSK_RX65N_2MB);
+
+	}
+
+	@Test
+	public void tc_02_CheckComponentView() throws Exception {
+		Utility.openSCFGEditor(projectModelSpecific);
+
+		bot.tree(1).getTreeItem("Startup").getNode("Generic").getNode("r_bsp").select();
+
+		bot.tree(1).getTreeItem(ProjectParameters.FolderAndFile.FOLDER_RTOS)
+				.getNode(ProjectParameters.FolderAndFile.FOLDER_RTOS_KERNEL)
+				.getNode(ProjectParameters.RTOSComponent.FREERTOS_KERNEL).select();
+
+		boolean checkConfigureView1 = bot.tree(2).getTreeItem("Configurations ").isVisible();
+
+		Utility.getProjectItemOnProjectExplorer(projectModelSpecific.getProjectName())
+				.contextMenu(MenuName.CONTEXT_MENU_CLOSE_PROJECT).click();
+		if (bot.activeShell().getText().equals(ProjectParameters.WINDOW_SAVE_RESOURCES)) {
+			bot.button(ButtonAction.BUTTON_DONT_SAVE).click();
+		}
+
+		Utility.getProjectItemOnProjectExplorer(projectModelSpecific.getProjectName())
+				.contextMenu(MenuName.CONTEXT_MENU_OPEN_PROJECT).click();
+		Utility.getProjectItemOnProjectExplorer(projectModelSpecific.getProjectName()).click();
+
+		Utility.openSCFGEditor(projectModelSpecific);
+		bot.tree(1).getTreeItem(ProjectParameters.FolderAndFile.FOLDER_RTOS)
+				.getNode(ProjectParameters.FolderAndFile.FOLDER_RTOS_KERNEL)
+				.getNode(ProjectParameters.RTOSComponent.FREERTOS_KERNEL).select();
+
+		boolean checkConfigureView2 = bot.tree(2).getTreeItem("Configurations ").isVisible();
+
+		if (!checkConfigureView1 || !checkConfigureView2) {
+			assertFalse(true);
+		}
+	}
+
+	@Test
+	public void tc_03_CheckDatalinkBetweenProperty() throws Exception {
```

Comment:
E: prevent hard-code property ID inside this method, same as methods below

Reply:
This will be fixed in latest commit